src/templates/flot/format-chart-def.ts

```diff
@@ -1,19 +1,20 @@
-import { IChartDef, ISingleYAxisMap, ISingleAxisMap, IAxisConfig } from "../../../src/chart-def";
-
-declare var moment: any;
-declare var numeral: any;
-
-"use strict";
```

Comment:
Removed `"use strict"` because it gets automatically produced by typescript's compiler